data/darktableconfig.xml.in

```diff
@@ -1773,6 +1773,20 @@
     <shortdescription>pixel interpolator</shortdescription>
     <longdescription>pixel interpolator used in rotation and lens correction (bilinear, bicubic, lanczos2, lanczos3).</longdescription>
   </dtconfig>
+  <dtconfig>
+    <name>plugins/lighttable/export/dimensions_type</name>
+    <type>int</type>
+    <default>0</default>
+    <shortdescription>unit of the printing size</shortdescription>
+    <longdescription>unit in which to input the image size.</longdescription>
+  </dtconfig>
+  <dtconfig>
+    <name>plugins/lighttable/export/print_dpi</name>
+    <type>int</type>
+    <default>300</default>
+    <shortdescription>DPI</shortdescription>
+    <longdescription>print resolusion in DPI.</longdescription>
+  </dtconfig>
   <dtconfig>
     <name>plugins/lighttable/export/width</name>
     <type>int</type>
```

Comment:
since recent changes in conf api, wouldn't it be good to define min (and max where applicable) for ints here? :)

Reply:
agreed, what would you propose ?

For DPI, we can probably limit from 50 to 4800.

For dimensions_type don't know, this control an enum in the code. So min=0 and max=? We may want to set max to last enum but in this case every time we add a new value one has to remember to increase the max here otherwise the value will always be clamped and between run dt won't keep the last selected value. So I'm not sure... We can of course set max to say 10 and be ok, but that's not really satisfying ! Any thoughts about this ?

Reply:
i think width/height should be min=0 with no max, dpi - I agree for min 50, but I don't know about max - I've seen printer manufacturers boasting 9600 dpi on some recent super-high-end models, so I think max 19200 would be "enough". the min for enum obviously would be 0, but for max - I think it could be generated but it's not really worth the effort to create generation code, so the code querying enum value could use `dt_conf_get_and_sanitize_int` kind of api with max defined as last element in enym?

Reply:
DPI min shoud be 72, maybe max could be 9600 but that looks like a marketing scam because photo labs always print at 300 DPI AFAIK.

Reply:
Thanks for feedback, I have added the constraints.